manifests/guestcluster/1.29/pvcsi.yaml

```diff
@@ -547,6 +547,7 @@ data:
   "tkgs-ha": "true"
   "cnsmgr-suspend-create-volume": "true"
   "workload-domain-isolation": "false"
+  "clean-up-protection-finalizer": "false"
```

Comment:
Can we name feature as "supervisor-pvc-and-snapshot-protection-finalizer"

Reply:
done